tools/mqtt-bench/bench.go

```diff
@@ -26,9 +26,14 @@ func Benchmark(cfg Config) {
 	var caByte []byte
 	if cfg.MQTT.TLS.MTLS {
 		caFile, err := os.Open(cfg.MQTT.TLS.CA)
-		defer caFile.Close()
+
+		defer func() {
+			if err = caFile.Close(); err != nil {
+				log.Printf("Could  not close file: %s", err)
+			}
+		}()
 		if err != nil {
-			fmt.Println(err)
+			log.Print(err)
```

Comment:
Why is this changed?

Reply:
Rodney left a comment saying this was a debug statement. But this has always been a print statement. So I changed it to log (so that it will give more infrormation when printed)